client/src/lib/components/Cards/CollectionCards/CollectionFullCard.tsx

```diff
@@ -93,36 +96,44 @@ const CollectionFullCard: React.FC<CollectionFullCardProps> = ({
   image,
   description,
   progressItems,
-  theme,
+  backgroundColorGradient,
+  textColor,
   onPress,
-}) => (
-  <Container onPress={onPress}>
-    <Gradient colors={theme?.backgroundColorGradient || ['transparent']}>
-      <Row>
-        <LeftColumn>
-          <TitleWrapper>
-            <IconWrapper>
-              <CollectionIcon fill={theme?.textColor} />
-            </IconWrapper>
+}) => {
+  const colors = useMemo(
+    () => backgroundColorGradient?.map(({color}) => color),
+    [backgroundColorGradient],
+  );
+
+  return (
+    <Container onPress={onPress}>
+      <Gradient colors={colors || ['transparent']}>
```

Comment:
Minor thing, but move `|| ['transparent']` inside the memo instead otherwise it's going to recreate it on every render